libnavui-dropin/src/main/java/com/mapbox/navigation/dropin/internal/factory/DropInUIMapboxNavigationFactory.kt

```diff
@@ -0,0 +1,24 @@
+package com.mapbox.navigation.dropin.internal.factory
```

Comment:
Developers will go looking for the classes they're going to use. If the root of the package is cluttered with classes they don't interact with it makes it more difficult to focus on what's relevant for them.  I think we should take as much out of the root package as possible. If the only class (or primary class) the developer interacts with is `NavigationView` and that's the only thing in the root package it's a lot simpler since there are no distractions.  

The classes in the interal.factory package are internal and they are factories. To me the naming is accurate.

Reply:
To me it makes sense to have an internal package when we cannot make class internal but we don't want developers to interact with it. In this case the factories inside `internal` package can be `internal class` itself and we should make them `internal`. Hence it doesn't make sense to add `internal` package. We can probably get away with just `factory` rather than `internal/factory`

Reply:
Agreed with @abhishek1508, these classes are not meant to be accessed from other modules, so they should live in the main package and be actually `internal`.